src/components/db.ts

```diff
@@ -148,6 +140,23 @@ const initRpsGameInfo = async (db: Database): Promise<void> => {
   );
 };
 
+const addSQLColumnIfNotExists = async (
```

Comment:
Can we add some documentation for this function? Explain what its purpose is, as well as what the different parameters are. We should specify that `columnDataType` is a SQL data type, and give an example (say "eg `VARCHAR(32)`").

Reply:
Got it. I'll add a small documentation comment for this.